tests/phpunit/tests/test-api.php

```diff
@@ -67,4 +79,92 @@ public function test_translated_objects_get_translation( $type ) {
 		$this->assertSame( 0, PLL()->model->$type->get_translation( $objects['en'], 'de' ) );
 		$this->assertSame( 0, PLL()->model->$type->get_translation( $objects['en'], 'chti' ) );
 	}
+
+	/**
+	 * @covers ::pll_insert_term
```

Comment:
Should we really start to use this annotation?

Reply:
As you see fit.
It's already done in this very class.
Here is the [documentation](https://docs.phpunit.de/en/10.5/annotations.html#covers) for some insight.

I stumbled upon that when doing a PR on WordPress, which uses that kind of annotation as a standard, so I thought "Why not?".

Reply:
Indeed I see that it's used in a few other places. I missed them when they were added (probably because I often don't review tests).

In my opinion it's misused. What it says is that your test covers only `pll_insert_term()`, which is wrong as it's more a test of `PLL_Translated_Term::insert()`.

I believe that this annotation is adapted for unit tests (or "pseudo" unit tests as a true unit test should not need it). It's certainly not adapted to wrappers like `pll_insert_term()`.

I never used it because most of the tests are integration tests and I like to know what lines are executed even indirectly.